arches/app/datatypes/datatypes.py

```diff
@@ -1427,6 +1471,23 @@ def transform_value_for_tile(self, value, **kwargs):
             compatible_renderers = self.get_compatible_renderers(tile_file)
             if len(compatible_renderers) == 1:
                 tile_file["renderer"] = compatible_renderers[0]
+
+            # if files include metadata, add metadata to the tile_file
+            localized_metadata_keys = {"altText", "attribution", "description", "title"}
+            languages = models.Language.objects.all()
+            language = get_language()
+
+            if isinstance(file_info, dict):
+                for key in localized_metadata_keys:
+                    val = file_info.get(key, "")
+                    if not isinstance(val, dict):
```

Comment:
We need to handle for cases when the key isn't included tile data:
```suggestion
                    if not isinstance(val, dict) or key not in tile_file.keys():
```